### ISSUE TITLE
[CLI] flag -h which stands for homepage doesn't work in new:project command

### ISSUE THREAD

**Issue by glammers1** [boostercloud/booster#195]
`-h` flag is duplicated for both help and homepage causing an error when trying to use -h as homepage.

```
$ boost new:project my-project -h "https://example.com"
create a new project from scratch

USAGE
  $ boost new:project [PROJECTNAME]

OPTIONS
  -a, --author=author                            who is writing this?
  -d, --description=description                  a short description
  -h, --help                                     show CLI help
  -h, --homepage=homepage                        the website of this project
  -l, --license=license                          which license will you use?
  -p, --providerPackageName=providerPackageName  package name implementing the cloud provider integration where the application will be deployed (i.e: "@boostercloud/framework-provider-aws"
  -r, --repository=repository                    the URL of the repository
  -v, --version=version                          the initial version

ExitError: EEXIT: 0
    at Object.exit (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/node_modules/@oclif/errors/lib/index.js:18:11)
    at Project.exit (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/node_modules/@oclif/command/lib/command.js:54:23)
    at Project._help (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/node_modules/@oclif/command/lib/command.js:127:21)
    at Object.parse (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/node_modules/@oclif/command/lib/flags.js:37:17)
    at Parser._flags (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/node_modules/@oclif/parser/lib/parse.js:149:42)
    at Parser.parse (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/node_modules/@oclif/parser/lib/parse.js:117:28)
    at Object.parse (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/node_modules/@oclif/parser/lib/index.js:27:27)
    at Project.parse (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/node_modules/@oclif/command/lib/command.js:86:41)
    at Project.runWithErrors (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/dist/commands/new/project.js:14:38)
    at Project.run (/Users/glammers/.nvm/versions/node/v12.16.3/lib/node_modules/@boostercloud/cli/dist/commands/new/project.js:11:21) {
  oclif: { exit: 0 },
  code: 'EEXIT'
}
```

The same command using the long version of the flag works:

```
$ boost new:project my-project --homepage "https://example.com"
? What's your project description? (default: "")
```

**Comment by dotalo** [comment]
This was solved with #216. You may close the issue :)

**Comment by juanjoman** [comment]
Closing this since it was [already solved](https://github.com/boostercloud/booster/blame/f18d2f850212bf84ca93f689f22bc32666985c7e/packages/cli/src/commands/new/project.ts#L31) by @MYF95. Now we are using `-H` instead of `-h`.
